src/middlewared/middlewared/plugins/iscsi_/alua.py

```diff
@@ -85,7 +85,7 @@ async def standby_enable_devices(self, devices):
             tochange = [extents[name] for name in devices]
             await self.middleware.call('iscsi.scst.set_devices_cluster_mode', tochange, 1)
             # We could expose the targets as we go along, but will just wait until the end.
-            # await self.middleware.call('service.reload', 'iscsitarget')
+            # await (await (await self.middleware.call('service.reload', 'iscsitarget')).wait(raise_error=True)).wait(raise_error=True)
```

Comment:
copy and replace bug. The comment is wrong.

Reply:
Wrong in what sense? Should it be just removed or changed?

Reply:
Why are there 3x `awaits`?